@app/admin/package.json

```diff
@@ -9,6 +9,63 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "sass": "^1.69.5"
+    "@auth/prisma-adapter": "^1.0.2",
```

Comment:
You want to install packages into the workspaces in which they are used, _not_ the root of the repository.

The root of the repository should contain dependencies that are important at the "repo" level. Things that exist _outside_ of the context of your applications.

Reply:
Yup, I understand that.

I put Auth in root because both the client and admin parts will use it. And I only needed sass for the admin part initially.